chia/util/initial-config.yaml

```diff
@@ -636,6 +641,14 @@ solver:
   # Enable or disable UPnP port forwarding
   enable_upnp: False
 
+  # Node IDs of trusted solver peers, only these can connect by default
+  trusted_peers:
+    localhost: 9999
```

Comment:
maybe this should use `*self_hostname` too. I think the rationale for that is that sometimes `localhost` refers to IPv6 which then fails.